eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -80,6 +70,10 @@ stages:
                         PackageName: ${{artifact.name}}
                         ServiceName: ${{parameters.ServiceDirectory}}
                         ForRelease: true
+                    - pwsh: |
```

Comment:
Given this is just calling a single script lets switch it to using the powershell tasks instead of the pwsh shortcut. See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/2/Azure-DevOps-Pipeline-Guidance?anchor=devops-tasks-vs-pwsh-script-shortcut for more info.